compiledb/cli.py

```diff
@@ -47,11 +48,14 @@ def __init__(self, infile, outfile, build_dir, exclude_files, no_build,
 
 
 @click.group(context_settings=CONTEXT_SETTINGS, invoke_without_command=True)
-@click.option('-p', '--parse', 'infile', type=click.File('r'),
+@click.option('-p', '--parse', 'logfile', type=click.File('r'),
               help='Build log file to parse compilation commands from.' +
               '(Default: stdin)', required=False, default=sys.stdin)
+@click.option('-i', '--input', 'infile',
+              help="Input file path (Default: compile_commands.json)",
+              required=False, default='compile_commands.json')
```

Comment:
I have a use case for keeping them separate for creating a build cache. If there are no changes between the input and output files, I delete the output file. Otherwise, I copy over the output file to the input file. Also, since the `-o / --output` parameter is set to a type of `File('w')`, can I read from that file and then write to that file? I'm not that familiar with the `click` package

Reply:
Actually, I can cover this use case by copying the current file to a new file. If they are the same, copy the new file over the current file, and delete the new file. Otherwise, just delete the new file.

I could have actually modified your package to check if anything has changed, but I didn't want to impact the performance of your package.